src/game_character.cpp

```diff
@@ -149,33 +148,60 @@ int Game_Character::GetScreenZ(bool apply_shift) const {
 	return z;
 }
 
-void Game_Character::Update() {
-	if (GetStopCount() >= GetMaxStopCount()) {
-		if (IsMoveRouteOverwritten()) {
-			MoveTypeCustom();
-		} else {
-			// Only events
+void Game_Character::UpdateMovement() {
+	bool was_move_route_overriden = IsMoveRouteOverwritten();
```

Comment:
> Move Route refactors
Is this unused? Can't find it in the diff

Reply:
Its unused, will remove